### PR TITLE
Add asciidoc NOTE: for community-supported plugins

### PR DESCRIPTION
**Pull request by kurtado**
This should read the list of Logstash-shipped plugins from the default_plugins.rb file and add an asciidoc NOTE: Admonition to signify that it is community-supported.

### CODE REVIEW

**Comment by kurtado** [comment]
@electrical would love your input as to how this would work with the generate-all-plugin-docs workflow.

File: docs/plugin-doc.asciidoc.erb [hunk 1]
```diff
@@ -2,6 +2,10 @@
 [[plugins-<%= section %>s-<%= name %>]]
 === <%= name %>
 
+<% unless default_plugin %>
+NOTE: This is a community-contributed plugin! It does not ship with logstash by default, but it is easy to install! To use this, you must have installed the contrib plugins package.
+<% end %>
```
Comment (kurtado) [inline docs/plugin-doc.asciidoc.erb]: Note text subject to update (no contrib :)

**Comment by suyograo** [comment]
@kurtado LGTM, please fix the message (removing contrib). see text here: https://github.com/elasticsearch/logstash/issues/2096

**Comment by suyograo** [comment]
LGTM

**Pull request closed without merging**